### ISSUE TITLE
Old migrations need to have `ActiveRecord::Migration` specified or will raise an error

### ISSUE THREAD

**Issue by carlosdanielpohlod** [openSUSE/osem#3099]
**I'm submitting a ..**
- [ x ] Bug Report
- [ ] Feature Request

**Current behavior:**
Currently, the specification of `ActiveRecord::Migration` in old migration files is required but is not specified, so `rails db:migrate` raises an error

**Expected correct behavior:**

**Steps to reproduce:**
run `rails db:migrate` when database is empty

**Other information:**

**Comment by carlosdanielpohlod** [comment]
I will take this issue to solve

**Comment by AndrewKvalheim** [comment]
See #2665 for a script that automates this.

**Comment by carlosdanielpohlod** [comment]
> See #2665 for a script that automates this.

You filled some files with the correct version, right? the other ones still do not have the version, can I fill in the others?

**Comment by AndrewKvalheim** [comment]
Yes, I skipped the migrations that I wasn’t taking the time to review, but ideally all should have the version specified.

**Comment by carlosdanielpohlod** [comment]
> Yes, I skipped the migrations that I wasn’t taking the time to review, but ideally all should have the version specified.

Ok

**Comment by hennevogel** [comment]
> Steps to reproduce:
> run rails db:migrate when database is empty

Just do `rails db:setup` instead...
